docs/crio.8.md

```diff
@@ -217,6 +218,8 @@ If `hooks_dir` is unset, CRI-O will currently default to `/usr/share/containers/
 
 **--version, -v**: Print the version
 
+**--version-file**="": Location for crio to lay down the version file.
```

Comment:
as much as I love a good ending period ('.'), remove the one here as it doesn't appear any other line has one.